src/components/InputForm.tsx

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react';
+import {
+  Box,
+  Button,
+  Input,
+  FormControl,
+  FormLabel,
+  FormHelperText,
+  Select,
+} from '@chakra-ui/react';
+
+interface Event {
```

Comment:
Personally, I think this should be named Absence

Reply:
fixed